arviz/plots/pairplot.py

```diff
@@ -34,6 +35,10 @@ def plot_pair(
     data : obj
         Any object that can be converted to an az.InferenceData object
         Refer to documentation of az.convert_to_dataset for details
+    group: Optional[str]
+        Specifies which InferenceData group should be plotted.  Defaults to 'posterior'.
+        Alternative values include 'prior' and any other strings used as dataset keys in the
+        InferenceData.
```

Comment:
I think the first line is enough, this last part may even confuse new users not fully understanding the relation between inference data and xarray datasets

Reply:
Okay, I'll do that in the next commit.